examples/vumps/vumps_2d_heisenberg.jl

```diff
@@ -42,12 +43,17 @@ function ITensorInfiniteMPS.unit_cell_terms(::Model"heisenberg2D"; width)
     opsum += -0.5, "S-", i, "S+", i + width
     opsum += "Sz", i, "Sz", i + width
   end
+  if yperiodic
+    opsum += -0.5, "S+", 1, "S-", width
+    opsum += -0.5, "S-", 1, "S+", width
```

Comment:
```suggestion
    opsum -= 0.5, "S+", 1, "S-", width
    opsum -= 0.5, "S-", 1, "S+", width
```
I see other code in the package avoids `-=`, I think it wasn't working in ITensors.jl at the time when I first wrote the package.